docs/topics/proc_creating-spring-boot-app.adoc

```diff
@@ -0,0 +1,209 @@
+[#proc_creating-a-spring-boot-app]
+= Creating a {SpringBoot} Application
+
+In addition to xref:mission-http-api-spring-boot-tomcat[using a booster], you can also create new {SpringBoot} applications from scratch and deploy them to OpenShift.
+
+== Creating an Application
+
+.Prerequisites
+* `mvn` installed.
```

Comment:
I would prefer _Maven_ instead of `mvn`. The latter is just the binary, whereas you need the complete build system.

Reply:
Agreed.  Besides, i think we use the full form in other similar instances in the docs